lib/student.rb

```diff
@@ -1 +1,19 @@
+class Student
+  attr_accessor :name, :age, :scores
 
+  def initialize(attributes)
+    @name = attributes[:name]
+    @age = attributes[:age]
+    @scores = []
+  end
+
+  def log_score(grade_number)
+    @scores << grade_number
+  end
+
+  def grade
+    average = @scores.sum do |score|
+      score.to_f / 2
+      end
```

Comment:
We don't want to do the division inside of the sum method. Also instead of hard-coding `2`, this should be dynamic.
```suggestion
  def grade
    average = @scores.sum do |score|
      score.to_f
    end
    average / @scores.count
  end
```